src/chat/PmConversationDetailsScreen.js

```diff
@@ -15,11 +15,14 @@ import { navigateToAccountDetails } from '../actions';
 import { getOwnUserId } from '../selectors';
 
 type Props = $ReadOnly<{|
-  navigation: AppNavigationProp<'group-details'>,
-  route: RouteProp<'group-details', {| recipients: PmKeyRecipients |}>,
+  navigation: AppNavigationProp<'pm-conversation-details'>,
+  route: RouteProp<'pm-conversation-details', {| recipients: PmKeyRecipients |}>,
 |}>;
 
-export default function GroupDetailsScreen(props: Props): Node {
+/**
+ * Details about a PM conversation, identified by PM key recipients.
```

Comment:
The second half of this is redundant with the type. Best to leave it out -- that way it doesn't become wrong if we change the type in the future (renaming it, or changing to something like `PmNarrow`, or something else.)